crates/networking/p2p/src/network/lean.rs

```diff
@@ -179,7 +196,50 @@ impl LeanNetworkService {
         event: SwarmEvent<ReamBehaviourEvent>,
     ) -> Option<ReamNetworkEvent> {
         match event {
-            SwarmEvent::Behaviour(_) => None,
+            SwarmEvent::Behaviour(ReamBehaviourEvent::Gossipsub(gossipsub_event)) => {
+                if let GossipsubEvent::Message { message, .. } = gossipsub_event {
+                    match LeanGossipsubMessage::decode(&message.topic, &message.data) {
+                        Ok(LeanGossipsubMessage::Block(signed_block)) => {
+                            if let Err(err) =
+                                self.chain_message_sender.send(LeanChainServiceMessage {
+                                    item: QueueItem::BlockItem((signed_block).data.clone()),
+                                })
+                            {
+                                warn!("failed to send block item to chain: {err:?}");
```

Comment:
```suggestion
                                warn!(
                                    "failed to send block for slot {} item to chain: {err:?}",
                                    signed_block.data.slot
                                );
```

I think we can log better here. Same comment for other `warn!` logs in three other places